lib/Window.js

```diff
@@ -54,6 +55,8 @@ module.exports = function Window(resp, windowObjects, innerWidth = 760, innerHei
     get: () => window
   });
 
+  Object.defineProperty(window, "localStorage", new Storage());
```

Comment:
This only supplies a getter, hence the property cannot be deleted. Makes it impossible to test a scenario without localStorage.

Add the following tests:
```js
    it("can be deleted for testing purposes", () => {
      delete window.localStorage;
      expect(window.localStorage).to.be.undefined;
    });

    it("can be overwritten for testing purposes", () => {
      const mockStorage = {};
      window.localStorage = mockStorage;
      expect(window.localStorage === mockStorage).to.be.true;
    });

    it("is enumerable", () => {
      expect(Object.keys(window)).to.include("localStorage");
    });
```